packages/runtime/src/next/api-handler/handlers/manifest.ts

```diff
@@ -28,5 +29,6 @@ export default async function handler(
     interactionMode: true,
     clientSideNavigation: true,
     elementFromPoint: false,
+    customBreakpoints: true,
```

Comment:
How come we need this information in the manifest? Is it for the builder to determine whether it should wait for the host to send breakpoints?